floor_generator/test/processor/entity_processor_test.dart

```diff
@@ -120,6 +120,50 @@ void main() {
     expect(actual.fields.length, equals(2));
   });
 
+  test('Ignore getter', () async {
+    final classElement = await _createClassElement('''
+      @entity
+      class Person {
+        @primaryKey
+        final int id;
+      
+        final String name;
+      
+        String get label => "" + id + ": " + name
+      
+        Person(this.id, this.name);
+        
+        static String foo = 'foo';
+      }
+    ''');
+
+    final actual = EntityProcessor(classElement).process();
+
+    expect(actual.fields.length, equals(2));
+  });
+
+  test('Ignore setter', () async {
+    final classElement = await _createClassElement('''
+      @entity
+      class Person {
+        @primaryKey
+        final int id;
+      
+        final String name;
+      
+        set printwith(String prefix) => print(prefix+name);
+
+        Person(this.id, this.name);
+        
+        static String foo = 'foo';
+      }
+    ''');
+
+    final actual = EntityProcessor(classElement).process();
+
+    expect(actual.fields.length, equals(2));
+  });
```

Comment:
💡 We could get rid of the static field and make the test a bit more explicit as shown in the following. Same applies to the "ignore getter" test.

```suggestion
    test('Ignore setter', () async {
      final classElement = await _createClassElement('''
      @entity
      class Person {
        @primaryKey
        final int id;
      
        final String name;
      
        set printwith(String prefix) => print(prefix+name);

        Person(this.id, this.name);
      }
    ''');

      final actual = EntityProcessor(classElement)
          .process()
          .fields
          .map((field) => field.name)
          .toList();

      expect(actual, equals(['id', 'name']));
    });
```